scripts/seed.py

```diff
@@ -98,6 +121,13 @@ def create_challenge(title, start_date, end_date, host_team):
 
 
 def create_challenge_phases(challenge, number_of_phases=1):
+    """
+    Creates a challenge phase and returns it.
+
+    Now create challenge phases for the above created challenge with parameters name,
```

Comment:
Replace with `Create challenge phases for the created challenges with parameters name, description, leaderboard_public, is_public, start_date, end_date, challenge, test_annotation, codename.`